subsys/bluetooth/audio/mcs.c

```diff
@@ -0,0 +1,989 @@
+/** @file
+ *  @brief Bluetooth Media Control Service - d09r01 IOP1 2019-08-22
```

Comment:
Isn't it version 1.0?

Reply:
It is.  Removed the whole version string - the spec is stable now, less need to track which version.